### DISCUSSION TITLE
For Inpaint, does it matter what model I use in the Base Model tab? Does the base model play any role?

### DISCUSSION THREAD

**Discussion by moviejimmy** [lllyasviel/Fooocus#1041] [Q&A]
I understand that Fooocus has its own inpainting model. So, what role does the Base Model play? Does it matter what I use there? I have tried a few models that they don't seem to affect much. Just want to know for sure.

Many thanks.

**Comment by lllyasviel** [comment]
Fooocus has its own inpainting **control** model, not inpainting model.
It control arbitary SDXL model to inpaint and does not influence the style of your base model.
This is not possible in other software now. Keep it as a secret and do not tell other people that you can do this in Fooocus!

**Reply by Phrixus2023** [reply]
The author has a great sense of humor.

**Reply by moviejimmy** [reply]
Thanks. To be honest, your inpainting is one of the best I have tried. I have a1111 and comfyui both and have used a lot of inpainting models, yours seems to be the most consistent.

What is the difference between Inpaint default and Modify Content? I am trying to change a dining table to different types. I use both default and Modify content and they seem to work just as well. Under what circumstances would they make a significant difference?
